tokio/src/runtime/io.rs

```diff
@@ -54,13 +50,11 @@ mod variant {
     pub(crate) type Driver = ParkThread;
 
     /// There is no handle
-    pub(crate) type Handle = ();
+    pub(crate) type Handle = Option<()>;
```

Comment:
Why is this switched to `Option`?

Reply:
Thanks! I've switched this back to `()`.